internal/querycoordv2/job/job_sync.go

```diff
@@ -89,12 +89,27 @@ func (job *SyncNewCreatedPartitionJob) Execute() error {
 		LoadPercentage: 100,
 		CreatedAt:      time.Now(),
 	}
-	err = job.meta.CollectionManager.PutPartition(job.ctx, partition)
+	err := job.meta.CollectionManager.PutPartition(job.ctx, partition)
 	if err != nil {
 		msg := "failed to store partitions"
 		log.Warn(msg, zap.Error(err))
 		return errors.Wrap(err, msg)
 	}
 
-	return nil
+	// manual trigger update next target
+	ready, err := job.targetObserver.UpdateNextTarget(req.GetCollectionID())
```

Comment:
Compared to before, awaiting the next target here, if the next target includes an accompanying segment change, the execution time here will be prolonged, potentially affecting the time taken by RootCoord to execute create_partition